### ISSUE TITLE
Safari 6.1.x - FireFox 29 - no dropdown menu

### ISSUE THREAD

**Issue by pepperstreet** [cwdx/tabs-to-dropdown#1]
Interesting project and effect, but it does not work in Safari 6.1.2 (webkit)
![bildschirmfoto 2014-05-07 um 01 27 54](https://cloud.githubusercontent.com/assets/1189521/2897009/f9090e0e-d576-11e3-818f-cb9d70cc2176.png)

**Comment by dmeijboom** [comment]
This is the expected behaviour, I've tested it in Safari and it works like it is supposed to work.
When your browser is to small for the menu all items that won't fit will become a dropdown menu.

**Comment by cwdx** [comment]
Hi Pepperstreet,

Thanks for checking out this project. Do you mean the dropdown menu is not showing up?

If so..

I've committed a couple of random performance fixes this afternoon which might fix this problem you're having.

Unfortunately I do not have Safari 6.1.2 available to me for testing purposes, so feel free to have another look and let me know if it's fixed.

**Comment by pepperstreet** [comment]
@dmeijboom 
Yes, I am aware of the purpose and functionality ;-) The issue is the MISSING drop-down. It simply does not display on click.

**Comment by pepperstreet** [comment]
@ChrisTheButcher 
Hi. Just updated to…
- Safari in OSX 10.8.5 to version **6.1.3 (8537.75.14)**
- FireFox 28 -> **29.0**

Same issue and behavior. **No** drop-down menu.

Only latest **Chrome 34.x** seems to work.
